config/config.go

```diff
@@ -0,0 +1,73 @@
+package config
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/BurntSushi/toml"
+)
+
+type Config struct {
+	HomeDir string
+	BinDir  string
+	VarDir  string
+	RunDir  string
+
+	Telemetry struct {
```

Comment:
nit: probably best to avoid anonymous structs like this. Easy enough to make a `TelemetryConfig` type.